### ISSUE TITLE
Auto Junk Frame for destroying only?

### ISSUE THREAD

**Issue by glassleo** [moody/Dejunk#152]
Could we please get some kind of option to only automatically open the junk frame if there is something to *destroy*?

I really don't need the junk frame to open when it's just auto selling stuff at a vendor. I really miss the old version where you could have different behavior for selling and destroying.

Thanks!

**Comment by zalidthebur** [comment]
I agree. The current behaviour is not good.
Having it auto-open at the mail, at bank, etc (due to bags auto-opening) is just annoying when there is nothing to destroy.
So yeah. Please have an option or default of only opening when there is something to destroy please :-)

**Comment by zalidthebur** [comment]
Why has this option changed from opening at merchants only to opening at merchants and when bags are opened ?
Without a check for destroyables, this is not user-friendly and I've lost a few items due to accidently hitting the destroy option when I thought it was the sell option.
I've had to turn off the Auto Junk frame option to stop this happening. Of course that means I've lost the the auto open at merchants which I like.
So please. Either add an option to only open at merchants, or only open when there is something on the destroy list present in the bag.
Also. Why are items that are for selling showing up in the destroy frame? Now I think about it. This may be the real issue. Since you now have one list for inclusions for selling and destroying. Unlike the old version of the addon which had separate lists.
Hopefully you can address this issue.

**Comment by glassleo** [comment]
Yeah, I agree and I think having a combined list is just problematic.

**Comment by zalidthebur** [comment]
Thanks for the update to this option :-)
